Runtime/Behaviours/CinemachineConfiner.cs

```diff
@@ -100,15 +101,36 @@ public bool IsValid
             }
         }
 
+        protected override void ConnectToVcam(bool connect)
+        {
+            base.ConnectToVcam(connect);
+
+            CinemachineVirtualCamera vcam = VirtualCamera as CinemachineVirtualCamera;
+            var components = vcam.GetComponentPipeline();
+            foreach (var component in components)
+            {
+                if (component.BodyAppliesAfterAim)
+                {
+                    var extraState = GetExtraState<VcamExtraState>(VirtualCamera);
+                    extraState.isFramingTransposer = true;
+                    break;
+                }
+            }
+        }
+        
+
         /// <summary>Callback to to the camera confining</summary>
         protected override void PostPipelineStageCallback(
             CinemachineVirtualCameraBase vcam,
             CinemachineCore.Stage stage, ref CameraState state, float deltaTime)
         {
             if (IsValid)
             {
-                // Move the body before the Aim is calculated
-                if (stage == CinemachineCore.Stage.Body)
+                var extraState = GetExtraState<VcamExtraState>(VirtualCamera);
```

Comment:
you have to use vcam here, not VirtualCamera.  See the correct call a few lines below.  Also, why get extraState twice?

Reply:
Totally true!